### ISSUE TITLE
RedisJSON v2.0.2 distribution size/checksum have been changed after tagging

### ISSUE THREAD

**Issue by osokin** [RedisJSON/RedisJSON#514]
name: Bug Report
description: File a bug report
title: "[Bug]: RedisJSON v2.0.2 distribution size/checksum have been changed after tagging"

Hi,

I've just found that RedisJSON v2.0.2 distribution has been re-rolled.
Could you please explain the release engineering procedure for the project.

Thank you.

**Comment by gkorland** [comment]
@osokin where did you get the binaries from ?

**Comment by osokin** [comment]
> @osokin where did you get the binaries from ?

@gkorland There're no binaries, I'm talking about the source code with v2.0.2 tag.

**Comment by gkorland** [comment]
@osokin you're right it seems like there is been a mistake in the way v2.0.2 was tagged so it was re-tagged. 
This is not a common practice but since this it's still in a RC phase the thought it's cleaner this way.

**Comment by osokin** [comment]
> @osokin you're right it seems like there is been a mistake in the way v2.0.2 was tagged so it was re-tagged. This is not a common practice but since this it's still in a RC phase the thought it's cleaner this way.

@gkorland Thanks for the update.

This still unclear, because if it's in R phase it's possible to use v2.0.x or v2.0.2rc then.  And why I've started this topic is because I committed an update to FreeBSD ports tree for version 2.0.2 with an assume that is release, but it's not.

That's why I'm interested what's release engineering process in the project.  And follow that I have a couple of questions.
How many changes are you going to add to the current state?  What's the release date for 2.0.2?
